src/settings/collectionsettingspage.ui

```diff
@@ -225,6 +217,13 @@ If there are no matches then it will use the largest image in the directory.</st
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QCheckBox" name="sort_skips_articles">
+        <property name="text">
+         <string>Skip articles such as &quot;The&quot; when sorting artist names</string>
```

Comment:
such as "the", "a" and "an"

Reply:
Is it fine if I drop “such as” if it’s going to be an exhaustive list? Or might the list be expanded later (for example to articles in other languages)?

Reply:
Since it's only 3 for now let's skip "such as".